### ISSUE TITLE
Carla ROS_Bridge

### ISSUE THREAD

**Issue by Te1moFernandes** [Te1moFernandes/PSA-P2-G3#9]
Instalar o carla ros bridge. Isto irá servir para receber informação recolhida pelo carla, e esta ser processada para voltar a ser enviada de volta de modo a ter o controlo automático por IA. A segunda parte ainda não se sabe como irá ser feita.

**Comment by miguelriemoliveira** [comment]
Para resolver o problem do MESA Intel fazer

    export VK_ICD_FILENAMES="/usr/share/vulkan/icd.d/nvidia_icd.json"

**Comment by miguelriemoliveira** [comment]
./CarlaUE4.sh -quality-level=Low -prefernvidia -RenderOffScreen

**Comment by miguelriemoliveira** [comment]
https://github.com/carla-simulator/ros-bridge/issues/593#issuecomment-1010581413

**Comment by miguelriemoliveira** [comment]
https://github.com/carla-simulator/ros-bridge/issues/593#issuecomment-1010581413

**Comment by Te1moFernandes** [comment]
Toda a gente conseguiu instalar o ros-bridge? @andrecabaco @ivodomingues @mafaldalemosmelo

**Comment by andrecabaco** [comment]
Sim, só estou com um problema em correr o CARLA com o ros-bridge, como fizeram na última aula. Só fizeste o que diz neste link, no ponto Run the ROS bridge?

https://carla.readthedocs.io/projects/ros-bridge/en/latest/ros_installation_ros1

**Comment by andrecabaco** [comment]
Boa tarde professor, estou com um problema quando dou "roslaunch carla_ros_bridge carla_ros_bridge.launch" para testar o funcionamento do ros bridge. Já instalei o ros bridge, com todos os passos que aparecem no guia https://carla.readthedocs.io/en/0.9.10/ros_installation/#run-the-ros-bridge
sendo que instalei pelo método B (repositório)
Para correr o roslaunch, corri todas as linhas que aparecem no guia, incluindo o source para o método B no terminal em que dou launch e aparece o seguinte no terminal:
REQUIRED process [carla_ros_bridge-2] has died!
process has died [pid 18472, exit code 127, cmd /home/pabundz/carla-ros-bridge/catkin_ws/src/ros-bridge/carla_ros_bridge/src/carla_ros_bridge/bridge.py __name:=carla_ros_bridge __log:=/home/pabundz/.ros/log/bba39ca6-ca3b-11ec-aba7-e7f3a2bcd165/carla_ros_bridge-2.log].
log file: /home/pabundz/.ros/log/bba39ca6-ca3b-11ec-aba7-e7f3a2bcd165/carla_ros_bridge-2*.log
Initiating shutdown!

Aconteceu algo semelhante ao professor?
@miguelriemoliveira

**Comment by mafaldalemosmelo** [comment]
eu estou com o mesmo erro que o @andrecabaco explicou em cima

**Comment by miguelriemoliveira** [comment]
Olá @andrecabaco e @mafaldalemosmelo ,

não sei o que poderá ser. Não me aconteceu. Sugiro que perguntem aos colegas para ver se conseguem ajudar, caso contrário tento eu ajudar na próxima aula depois das apresentações.
